File: tuan2/tuan22MainActivity.java
package com.example.firstapp.tuan2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import com.example.firstapp.R;

public class tuan22MainActivity extends AppCompatActivity {
    ListView lv;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tuan22_main);
        lv=findViewById(R.id.tuan22ListView);
        //nguon du lieu
        String[] arr =new String [] {"Mon 1","Tuan 2","mon 2","mon 3","mon 4","mon 5","tuan 6","aaa","bbb"};

        ArrayAdapter<String> adapter = new ArrayAdapter<>( this, android.R.layout.simple_list_item_1,arr);
        //hien thi
        lv.setAdapter(adapter);
    }
}

File: tuan22/tuan22MainActivity.kt
package com.example.firstapp.tuan22

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity
import com.example.firstapp.R

class tuan22MainActivity : AppCompatActivity() {
    var txt1: EditText? = null
    var txt2:EditText? = null
    var btn1: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tuan22_main2)

        txt1 = findViewById(R.id.tuan22txt1)
        txt2 = findViewById(R.id.tuan22txt2)
        btn1 = findViewById(R.id.tuan22btn1)

        btn1!!.setOnClickListener(View.OnClickListener {
            var i=Intent(this@tuan22MainActivity,
                tuan22secondActivity::class.java)
            i.putExtra("so1",txt1!!.text.toString())
            i.putExtra("so2",txt2!!.text.toString())
            startActivity(i)

        })
    }
}